### ISSUE TITLE
Unable to add add-on to HA

### ISSUE THREAD

**Issue by dewildep** [pssc/ha-addon-squeezelite#20]
Hello, I came across this add-on for HA.
I used to button in the code page on GitHub to add it to my HA install. The repo is added but I can't find the plug-in to add it. I search for "squeeze" but unable to find it.
Can you help me out? Thx
![Screenshot_20241218-230115](https://github.com/user-attachments/assets/cec71c44-674e-4f4c-a53d-dd0509faede7)

**Comment by pssc** [comment]
At a guess try again later it sometimes takes some time to pull the lis tof add-ons available from a new repo, but do goy to the addon repo list and check its there and then the addon store...

**Comment by pssc** [comment]
looks like you have added the LMS one not the squeezelite one there in different repos

**Comment by dewildep** [comment]
ok, so aparently i took the correct repo but i first had to enable "user profile" "General" "Advanced Mode"
![image](https://github.com/user-attachments/assets/3851669d-c22c-45da-a182-c768890c7b47)
After enabeling it i was able to see the addon:
![image](https://github.com/user-attachments/assets/0e73eb7a-6df2-4a95-853e-eed51a5427c8)

Any reason why there are 9 clones?

Am i correct that i can use this plugin to control my existing Squeeze players in my home? I have 5 of them so integration in HA to control them would be cool.
Or is this plugin only meant to play music on the same device as where HA is installed?

**Comment by pssc** [comment]
So for control you want the squeezebox integration for control assuming your using LMS s this plugin only meant to play music on the same device as where HA is installed supporting up to 10 output devices which should be enough for the more esoteric setups.

**Comment by pssc** [comment]
Do you know why you had to enable Advanced mode? I should either note that or fix it

**Comment by dewildep** [comment]
> So for control you want the squeezebox integration for control assuming your using LMS s this plugin only meant to play music on the same device as where HA is installed supporting up to 10 output devices which should be enough for the more esoteric setups.

Ok, that is clear now for me.
I already have LMS running on a Synology NAS with several hardware players connected (Squeezebox touch, PiCorePlayer, SqueezePlay on Windows client)
Great that you support up to 10 output devices with your add-on ! But not what i was looking for ;-)

i'll go ahead with https://www.home-assistant.io/integrations/squeezebox to control my existing players from HA.

**Comment by dewildep** [comment]
> Do you know why you had to enable Advanced mode? I should either note that or fix it

Yes, i needed to enable Advanced mode in order to be able to see the SqueezeLite icons appear in HA.
As you can see in my previous post, i did not see anything after adding the add-on. So maybe you will have to look into this.

So i added it via the button on this github page:
![image](https://github.com/user-attachments/assets/37ecaf37-bb82-421f-84e5-b07b03fe7c81)
![image](https://github.com/user-attachments/assets/cc6f59f7-4d5b-421c-b173-6436c2aea4c9)
![image](https://github.com/user-attachments/assets/27c7f289-9be9-452e-96a0-2f774c4f0d68)

After adding it i could not find it back in my add-on page. I also restarted HA but this did not help.
I could see that the repository was added:
![image](https://github.com/user-attachments/assets/40f39f9d-b7ba-4d01-adcc-7e7f3d8cd50c)

I then noticed a message on the bottom of the "Add-on Store"
**Missing add-ons? Enable advanced mode in your user profile page**
![image](https://github.com/user-attachments/assets/e27991e7-50ab-423f-88c5-9cd68aff6592)
I enabled it:
![image](https://github.com/user-attachments/assets/461b2e45-706d-4fd0-a5b3-17f3b19ef113)
Went back to the Add-On Store and tried again. Now i was able to see them:
![image](https://github.com/user-attachments/assets/adead18c-06b5-4f48-ad7b-8f337649ebcc)
